From f04ca743c0658941bba3e439838f1e45b7ff407b Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Thu, 4 Apr 2019 14:47:29 +0100
Subject: [PATCH 4/4] Bug fix for incomplete PowerLineFrequency enumeration map

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 drivers/media/usb/uvc/uvc_ctrl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/usb/uvc/uvc_ctrl.c b/drivers/media/usb/uvc/uvc_ctrl.c
index c2ad102..4621e5c 100644
--- a/drivers/media/usb/uvc/uvc_ctrl.c
+++ b/drivers/media/usb/uvc/uvc_ctrl.c
@@ -358,6 +358,7 @@ static struct uvc_menu_info power_line_frequency_controls[] = {
 	{ 0, "Disabled" },
 	{ 1, "50 Hz" },
 	{ 2, "60 Hz" },
+	{ 3, "Auto" },
 };
 
 static struct uvc_menu_info exposure_auto_controls[] = {
-- 
2.7.4

