From 6ce15a06e1ba02ebbc57c07e4269c473584ae6c3 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Wed, 5 Apr 2017 19:04:21 +0100
Subject: [PATCH] Remove R200 fix from udev rules

---
 config/99-realsense-libusb.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/99-realsense-libusb.rules b/config/99-realsense-libusb.rules
index d74138a..0ded643 100644
--- a/config/99-realsense-libusb.rules
+++ b/config/99-realsense-libusb.rules
@@ -1,5 +1,5 @@
 # RealSense UVC cameras (R200, F200, SR300 LR200, ZR300)
-SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev", RUN+="/usr/local/bin/usb-R200-in_udev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a66", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0aa5", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0abf", MODE:="0666", GROUP:="plugdev"
-- 
1.9.1

