From 23df9327ec2dc68dc575cec22fa6eec3b8a12d37 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Fri, 13 Jun 2025 12:52:19 +0100
Subject: [PATCH 1/5] Enable riscv arch compilation
Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 CMake/unix_config.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/unix_config.cmake b/CMake/unix_config.cmake
index 5a187c7c7..c97826610 100644
--- a/CMake/unix_config.cmake
+++ b/CMake/unix_config.cmake
@@ -28,6 +28,7 @@ macro(os_set_flags)
     elseif(${MACHINE} MATCHES "powerpc64(le)?-linux-gnu")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -ftree-vectorize")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ftree-vectorize")
+    elseif(${MACHINE} MATCHES "riscv*")
     else()
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mssse3")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
-- 
2.43.0

