From 8eee83c68e99f15d09d6aac8ad933b84a86a2700 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Fri, 13 Jun 2025 12:53:20 +0100
Subject: [PATCH 2/5] Avoid installing viewer presets
Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 tools/realsense-viewer/CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/tools/realsense-viewer/CMakeLists.txt b/tools/realsense-viewer/CMakeLists.txt
index 4e6c1768e..f3728b610 100644
--- a/tools/realsense-viewer/CMakeLists.txt
+++ b/tools/realsense-viewer/CMakeLists.txt
@@ -252,9 +252,4 @@ install(
     TARGETS ${PROJECT_NAME}
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
 )
-#https://cmake.org/cmake/help/latest/command/install.html
-install(DIRECTORY presets/
-    DESTINATION $ENV{HOME}/Documents/librealsense2/presets
-    FILES_MATCHING PATTERN "*.preset"
-)
 endif()
-- 
2.43.0

