From f6a5595b394c95b454cb691c1a65532da55daf5d Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Fri, 13 Jun 2025 12:54:03 +0100
Subject: [PATCH 3/5] Remove libusb from targets list
Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 CMake/libusb_config.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMake/libusb_config.cmake b/CMake/libusb_config.cmake
index ca221ca66..f38e38f73 100644
--- a/CMake/libusb_config.cmake
+++ b/CMake/libusb_config.cmake
@@ -10,5 +10,4 @@ if (NOT TARGET usb)
     else()
         include(CMake/external_libusb.cmake)
     endif()
-    install(TARGETS usb EXPORT realsense2Targets)
 endif()
-- 
2.43.0

