From 23aa75f6f368da4b8b27eccb38776193e3f01a80 Mon Sep 17 00:00:00 2001
From: Ross Lannen <ross.lannen@gmail.com>
Date: Wed, 5 Feb 2020 12:27:29 -0700
Subject: [PATCH] Fix arch detection on linux

---
 CMake/unix_config.cmake | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/CMake/unix_config.cmake b/CMake/unix_config.cmake
index 2ecd1ed02..f67419141 100644
--- a/CMake/unix_config.cmake
+++ b/CMake/unix_config.cmake
@@ -6,11 +6,10 @@ macro(os_set_flags)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -g -Wno-missing-field-initializers")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-switch -Wno-multichar -Wsequence-point -Wformat -Wformat-security")
 
-    execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpmachine OUTPUT_VARIABLE MACHINE)
-    if(${MACHINE} MATCHES "arm-linux-gnueabihf")
+    if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
-    elseif(${MACHINE} MATCHES "aarch64-linux-gnu")
+    elseif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mstrict-align -ftree-vectorize")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mstrict-align -ftree-vectorize")
     elseif(${MACHINE} MATCHES "powerpc64(le)?-linux-gnu")
@@ -20,7 +19,7 @@ macro(os_set_flags)
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mssse3")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
         set(LRS_TRY_USE_AVX true)
-    endif(${MACHINE} MATCHES "arm-linux-gnueabihf")
+    endif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
 
     if(BUILD_WITH_OPENMP)
         find_package(OpenMP REQUIRED)
-- 
2.17.1

