From 90f6e255e45dcd0e774c7d0995a5519c9712a2a3 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Mon, 8 Jun 2020 10:56:57 +0100
Subject: [PATCH 1/3] Fix arch detection on linux

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 CMake/unix_config.cmake | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/CMake/unix_config.cmake b/CMake/unix_config.cmake
index 2ecd1ed..50dd01e 100644
--- a/CMake/unix_config.cmake
+++ b/CMake/unix_config.cmake
@@ -6,21 +6,20 @@ macro(os_set_flags)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -g -Wno-missing-field-initializers")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-switch -Wno-multichar -Wsequence-point -Wformat -Wformat-security")
 
-    execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpmachine OUTPUT_VARIABLE MACHINE)
-    if(${MACHINE} MATCHES "arm-linux-gnueabihf")
+    if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
-    elseif(${MACHINE} MATCHES "aarch64-linux-gnu")
+    elseif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mstrict-align -ftree-vectorize")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mstrict-align -ftree-vectorize")
-    elseif(${MACHINE} MATCHES "powerpc64(le)?-linux-gnu")
+    elseif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "powerpc64")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -ftree-vectorize")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ftree-vectorize")
     else()
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mssse3")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
         set(LRS_TRY_USE_AVX true)
-    endif(${MACHINE} MATCHES "arm-linux-gnueabihf")
+    endif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
 
     if(BUILD_WITH_OPENMP)
         find_package(OpenMP REQUIRED)
-- 
2.7.4

