From e3151a52558c6882003d69602c6ebc86ff47c35a Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Mon, 8 Jun 2020 10:58:12 +0100
Subject: [PATCH 2/3] Remove R200 fix from udev rules

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 config/99-realsense-libusb.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/99-realsense-libusb.rules b/config/99-realsense-libusb.rules
index 0144322..e3eb567 100644
--- a/config/99-realsense-libusb.rules
+++ b/config/99-realsense-libusb.rules
@@ -2,5 +2,5 @@
 # Device rules for Intel RealSense devices (R200, F200, SR300 LR200, ZR300, D400, L500, T200)
-SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev", RUN+="/usr/local/bin/usb-R200-in_udev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a66", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0aa5", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0abf", MODE:="0666", GROUP:="plugdev"
-- 
2.7.4

