From dd470e89d5635b3b94e1bfdecc5b51cdf161b8d5 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Tue, 8 Oct 2019 12:08:04 +0100
Subject: [PATCH 2/2] Remove R200 fix from udev rules

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 config/99-realsense-libusb.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/99-realsense-libusb.rules b/config/99-realsense-libusb.rules
index 30f522d..8b6e96a 100644
--- a/config/99-realsense-libusb.rules
+++ b/config/99-realsense-libusb.rules
@@ -1,5 +1,5 @@
 # Intel RealSense UVC cameras (R200, F200, SR300 LR200, ZR300, D400, L500)
-SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev", RUN+="/usr/local/bin/usb-R200-in_udev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a66", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0aa5", MODE:="0666", GROUP:="plugdev"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0abf", MODE:="0666", GROUP:="plugdev"
-- 
2.7.4

