From 0bd532ea7069f71c1cb50d0d7734acdb4eb0747a Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Wed, 26 Jan 2022 15:58:21 +0000
Subject: [PATCH 3/3] Avoid installing viewer presets

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 tools/realsense-viewer/CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/tools/realsense-viewer/CMakeLists.txt b/tools/realsense-viewer/CMakeLists.txt
index 1ca7b0d..d087f0a 100644
--- a/tools/realsense-viewer/CMakeLists.txt
+++ b/tools/realsense-viewer/CMakeLists.txt
@@ -260,9 +260,4 @@ install(
     TARGETS realsense-viewer
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
 )
-#https://cmake.org/cmake/help/latest/command/install.html
-install(DIRECTORY presets/
-    DESTINATION $ENV{HOME}/Documents/librealsense2/presets
-    FILES_MATCHING PATTERN "*.preset"
-)
 endif()
-- 
2.7.4

