From 09e86de0b340dc3905d25ffe1f17d97ea5ab8651 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Tue, 25 Jan 2022 22:11:54 +0000
Subject: [PATCH] Enable riscv arch compilation

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 CMake/unix_config.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/unix_config.cmake b/CMake/unix_config.cmake
index 4ec81bbfa..6fa17da62 100644
--- a/CMake/unix_config.cmake
+++ b/CMake/unix_config.cmake
@@ -16,6 +16,7 @@ macro(os_set_flags)
     elseif(${MACHINE} MATCHES "powerpc64(le)?-linux-gnu")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -ftree-vectorize")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ftree-vectorize")
+    elseif(${MACHINE} MATCHES "riscv*")
     else()
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mssse3")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
