From 0b240f1def08da222a921d720697131aca0d7e54 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Mon, 15 Jul 2019 15:11:38 +0100
Subject: [PATCH] Fix arch detection on linux

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 CMake/unix_config.cmake | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/CMake/unix_config.cmake b/CMake/unix_config.cmake
index ba9210b..6457766 100644
--- a/CMake/unix_config.cmake
+++ b/CMake/unix_config.cmake
@@ -5,18 +5,17 @@ macro(os_set_flags)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -pedantic -g -Wno-missing-field-initializers")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-switch -Wno-multichar -Wsequence-point -Wformat-security")
 
-    execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpmachine OUTPUT_VARIABLE MACHINE)
-    if(${MACHINE} MATCHES "arm-linux-gnueabihf")
+    if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mfpu=neon -mfloat-abi=hard -ftree-vectorize")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfpu=neon -mfloat-abi=hard -ftree-vectorize")
-    elseif(${MACHINE} MATCHES "aarch64-linux-gnu")
+    elseif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mstrict-align -ftree-vectorize")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mstrict-align -ftree-vectorize")
     else()
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mssse3")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
         set(LRS_TRY_USE_AVX true)
-    endif(${MACHINE} MATCHES "arm-linux-gnueabihf")
+    endif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
 
     if(BUILD_WITH_OPENMP)
         find_package(OpenMP REQUIRED)
-- 
2.7.4

