From a417d6c2975d0227280003a716462b89da41ac44 Mon Sep 17 00:00:00 2001
From: Scott Ware <scott.r.ware@intel.com>
Date: Fri, 5 Apr 2019 10:09:08 +0100
Subject: [PATCH] Use system GLFW library

Signed-off-by: Scott Ware <scott.r.ware@intel.com>
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fa0f58f..ad31087 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,10 +49,6 @@ option(BUILD_CV_KINFU_EXAMPLE "Build OpenCV KinectFusion example" OFF)
 
 add_subdirectory(wrappers)
 
-if (BUILD_EXAMPLES AND BUILD_GRAPHICAL_EXAMPLES)
-    add_subdirectory(third-party/glfw)
-endif()
-
 if(BUILD_EXAMPLES)
     add_subdirectory(examples)
     add_subdirectory(tools)
-- 
2.7.4

